RELEASE.md

```diff
@@ -23,15 +23,15 @@ additionally.
 
 ## Release Tags and Branches
 
-Releases are tagged with tags like `X.Y.Z`, e.g. `1.0.2`. A special format like
-`1.0.0-rc.2` is a pre-release. It is the second release candidate before
-releasing the final `1.0.0`.
+Releases are tagged with tags like `v.X.Y.Z`, e.g. `v.1.0.2`. A special format like
```

Comment:
I think it's more common to tag like vX.Y.Z, can you confirm?

Reply:
Yes, will updated this.